### DISCUSSION TITLE
yarn과 npm의 차이는?

### DISCUSSION THREAD

**Discussion by rladpwl0512** [woowacourse-study/2022-woowahan-bansanghwe#34] [JavaScript]
npm, yarn 모두 자바스크립트 패키지 매니저이다. 

yarn은  npm의 부족한 부분을 개선하기위해 facebook에서 개발한 것이다. 기본적으로 npm 구조를 따르지만, 설치 속도, 보안적으로 개선된 부분들이 있다. 

하지만, yarn은 용량을 더 많이 잡아먹고, npm도 점점 속도가 빨라지고 있기 때문에 취향껏, 팀 컨벤션에 따라 사용하면 된다. 

---
- reference 
https://yarnpkg.com/

**Comment by InKyoJeong** [comment]
에지님은 어떤걸 선호하시나요??

**Reply by rladpwl0512** [reply]
`npm`을 사용했을 때의 큰 성능 저하(설치 속도 저하)를 느낄만큼 큰 프로젝트를 진행하지 않았고, `npm`이 아직 익숙하기도 해서 `npm`을 선호합니다 :)
